### PR TITLE
#1 - added ability for pods

### PR DESCRIPTION
**Pull request by kentmw**

### CODE REVIEW

**Comment by arikwex** [comment]
Can this be more of a "podify" or "set paths" function?

setPaths() // no input would do what you've provided.
setPaths({app: 'codez'}) // would override params.

File: index.js [hunk 1]
```diff
@@ -180,6 +180,16 @@ module.exports = function(gulp) {
 
     nonResources: function(nrsc) {
       nonResources = nrsc;
+    },
+
+    setAsPods: function() {
```
Comment (mandragorn) [inline index.js]: I don't understand how this addresses pods.

File: index.js [hunk 2]
```diff
@@ -180,6 +182,19 @@ module.exports = function(gulp) {
 
     nonResources: function(nrsc) {
       nonResources = nrsc;
+    },
+
+    configurePaths: function(options) {
+
+      console.log(options);
```
Comment (arikwex) [inline index.js]: please remove console logs

File: index.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 module.exports = function(gulp) {
   var browserifyBundler, copyResources, copyResource, buildStyles, buildApp, configureBrowserify,
     transformStack = [],
+    styleIncludePaths = ['.'],
```
Comment (arikwex) [inline index.js]: random '.'?

**Pull request merged**